### PR TITLE
Add tracking to org member page

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner @adamjgrant

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileInviteSearch.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ angular.module('kifi')
           if (scope.share.message) {
             opts.message = scope.share.message;
           }
-          //return libraryService.shareLibrary(scope.library.id, opts);
-          return orgProfileService.sendOrgMemberInvite(scope.organization.id, opts);
+          var cb = scope.modalData.returnAction;
+          return orgProfileService.sendOrgMemberInvite(scope.organization.id, opts).then(cb);
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileInviteSearch.js]: why assign this to `cb`?

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileInviteSearch.js]: I mean, as opposed to just putting the function in the `then`. Seems to go against the idea behind promises.

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileInviteSearch.js]: Hmm good point, I thought `cb` would communicate my intent more effectively, but looking at it with fresh eyes, `.then(scope.modalData.returnAction)` looks better

**Comment by andrewconner** [comment]
Looks good to me

**Comment by adamjgrant** [comment]
Late to the party but good stuff :+1:

**Comment by cvializ** [comment]
It's ok, there wasn't any :cake: so you didn't miss much

**Pull request merged**